tests/ops/qubit/test_matrix_ops.py

```diff
@@ -576,6 +578,114 @@ def test_matrix_already_in_cache_list(self, mat, op):
         assert len(cache["matrices"]) == 3
 
 
+class TestBlockEncode:
+    @pytest.mark.parametrize(
+        ("input_matrix", "wires"),
+        [
+            (1, 1),
+            ([1], 1),
+            ([[1]], 1),
+            (np.array(1), [1]),
+            (np.array([1]), 1),
+            (np.array([[1]]), 1),
+            ([[1, 0], [0, 1]], [0, 1]),
+            (np.array([[1, 0], [0, 1]]), range(2)),
+            (np.identity(3), ["a", "b", "c"]),
+        ],
+    )
+    def test_accepts_various_types(self, input_matrix, wires):
+        qml.BlockEncode(input_matrix, wires)
```

Comment:
I think you should still have an assert statement here, maybe something like: 
`assert op.parameters == input_matrix` and `assert op.hyperparameters == expected_hyperparameters`. This is to make sure that you arnt just constructing a random op, but it is kind of what you expect it to be.